server/db/migrations/19_volume_migration.rb

```diff
@@ -0,0 +1,25 @@
+
+class VolumeMigration < Mongodb::Migration
+
+  extend VolumesHelpers
+
+  def self.up
+    Volume.create_indexes
+
+    GridService.each do |service|
+      service.volumes.each do |v|
+        volume_spec = parse_volume(v)
```

Comment:
There might be services with invalid volume specs. Not much we can do about those, but maybe not explode the entire mutation?